pilot/pkg/model/extensions.go

```diff
@@ -136,16 +214,12 @@ func convertToWasmPluginWrapper(originPlugin config.Config) *WasmPluginWrapper {
 		return nil
 	}
 
-	cfg := &anypb.Any{}
 	if wasmPlugin.PluginConfig != nil && len(wasmPlugin.PluginConfig.Fields) > 0 {
-		cfgJSON, err := protomarshal.ToJSON(wasmPlugin.PluginConfig)
+		_, err := protomarshal.ToJSON(wasmPlugin.PluginConfig)
```

Comment:
a bit confused what the point of this is

Reply:
It is only to validate that plugin config is correct. We build it else where.

Reply:
Why don't we store `*wasmextensions.PluginConfig` still? Then we can compute it once (like before), and don't have to validate it twice (slow/error prone)

Reply:
We mutate the pluginConfig later.  Let me see if that can be improved in the follow-up